src/NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp

```diff
@@ -24,6 +24,14 @@ struct LogicalImpl;
 /// \ingroup NumericalAlgorithmsGroup
```

Comment:
Commit message: "Document template parameter of logical and partial_derivatives"